pkgs/servers/home-assistant/custom-lovelace-modules/hourly-weather/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "hourly-weather";
+  version = "6.1.0";
+
+  src = fetchFromGitHub {
+    owner = "decompil3d";
+    repo = "lovelace-hourly-weather";
+    rev = version;
+    hash = "sha256-IvwWeksEnq44PhCpUXVufYb8w5XEX9Dk0LMLLeM7Dps=";
+  };
+
+  npmDepsHash = "sha256-4uQfaYMg533xZobAl6+/9+FsHAFBm03wrF01nDa53Qg=";
+
+  env.CYPRESS_INSTALL_BINARY = "0";
```

Comment:
I am not 100% sure about this, I picked it out of another derivation using `buildNpmPackage` and it helped the failing build...

Reply:
I think that is fine.